### PR TITLE
more subtle color approach on project areas

### PR DESCRIPTION
**Pull request by pflopez**
Screen Shot 2023-11-10 at 18 29 08

### CODE REVIEW

File: src/interface/src/app/plan/project-areas/project-areas.component.html [hunk 1]
```diff
@@ -31,10 +31,8 @@ <h3 class="title">Project Areas</h3>
     </tr>
   </thead>
   <tbody>
-    <tr
-      *ngFor="let area of areas"
-      [style.background-color]="colors[area.id - 1] + '4D'">
-      <td class="area-id">
+    <tr *ngFor="let area of areas">
+      <td class="area-id" [style.border-left-color]="colors[area.id - 1]">
```
Comment (lastminutediorama) [inline src/interface/src/app/plan/project-areas/project-areas.component.html:35]: I should have asked this before, but is there any need to bounds check this?

**Review by lastminutediorama — APPROVED** [review]

File: src/interface/src/app/plan/project-areas/project-areas.component.html (hunk 1 above)
Comment (pflopez) [reply src/interface/src/app/plan/project-areas/project-areas.component.html]: Good call on this! Ive updated to use the function `getColorForProjectPosition` which cycles thru the colors, and added tests and edge cases. Thanks!
